pmg/templates/admin/my_edit.html

```diff
@@ -8,7 +8,7 @@
         <a href="{{ admin_view.alert_url(model) }}" class="btn btn-default"><i class="fa fa-envelope"></i> Send alert</a>
         {% endif %}
 	{% if admin_view.name == "Committees" %}
-	<script defer src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Why are we deleting defer here?

Reply:
The jquery was not loading in time, good ol' $ is not defined.

Found a better way to do it.